Failed: JmsConnectorsTest.publishAndConsumeJmsTextMessagesWithClientAcknowledgement

```
[error] Test akka.stream.alpakka.jms.javadsl.JmsConnectorsTest.publishAndConsumeJmsTextMessagesWithClientAcknowledgement failed: java.util.concurrent.ExecutionException: javax.jms.IllegalStateException: The Session is closed, took 0.036 sec
[error]     at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:357)
[error]     at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1915)
[error]     at scala.concurrent.java8.FuturesConvertersImpl$CF.super$get(FutureConvertersImpl.scala:84)
[error]     at scala.concurrent.java8.FuturesConvertersImpl$CF.$anonfun$get$2(FutureConvertersImpl.scala:84)
[error]     at scala.concurrent.BlockContext$DefaultBlockContext$.blockOn(BlockContext.scala:53)
[error]     at scala.concurrent.package$.blocking(package.scala:142)m
[error]     at scala.concurrent.java8.FuturesConvertersImpl$CF.get(FutureConvertersImpl.scala:84)
[error]     at akka.stream.alpakka.jms.javadsl.JmsConnectorsTest.lambda$publishAndConsumeJmsTextMessagesWithClientAcknowledgement$12(JmsConnectorsTest.java:551)
[error]     at akka.stream.alpakka.jms.javadsl.JmsConnectorsTest.withServer(JmsConnectorsTest.java:730)
[error]     at akka.stream.alpakka.jms.javadsl.JmsConnectorsTest.publishAndConsumeJmsTextMessagesWithClientAcknowledgement(JmsConnectorsTest.java:514)
[error]     ...
[error] Caused by: javax.jms.IllegalStateException: The Session is closed
[error]     at org.apache.activemq.ActiveMQSession.checkClosed(ActiveMQSession.java:771)
[error]     at org.apache.activemq.ActiveMQMessageConsumer$1.execute(ActiveMQMessageConsumer.java:605)
[error]     at org.apache.activemq.command.ActiveMQMessage.acknowledge(ActiveMQMessage.java:102)
[error]     at akka.stream.alpakka.jms.javadsl.JmsConnectorsTest.lambda$null$a934e49e$1(JmsConnectorsTest.java:545)
[error]     at akka.stream.javadsl.Source.$anonfun$map$1(Source.scala:1072)
[error]     at akka.stream.impl.fusing.Map$$anon$9.onPush(Ops.scala:51)
[error]     at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:519)
...
```

https://travis-ci.org/akka/alpakka/jobs/383090430

--- a/jms/src/test/java/akka/stream/alpakka/jms/javadsl/JmsConnectorsTest.java
+++ b/jms/src/test/java/akka/stream/alpakka/jms/javadsl/JmsConnectorsTest.java
@@ -491,13 +491,13 @@ public class JmsConnectorsTest {
           // #run-jms-source-with-ack
           CompletionStage<List<String>> result =
               jmsSource
-                  .take(msgsIn.size())
                   .map(
                       message -> {
                         String text = ((ActiveMQTextMessage) message).getText();
                         message.acknowledge();
                         return text;
                       })
+                  .take(msgsIn.size())
                   .runWith(Sink.seq(), materializer);
           // #run-jms-source-with-ack
 

